src/main/content/_assets/js/collections.js

```diff
@@ -1,82 +1,155 @@
-$(document).ready(function() {
-    fetchAllInstances()
-        .then(setInstanceSelections)
-        .then(handleInitialCLIAuth)
-        .then(handleCollectionsRequests)
-    //TODO then select/fetch current instance from url param for the dropdown
```

Comment:
since `hideAndEmptyTable` might not be necessary at this point - we can put this back